profiles/default/package.json

```diff
@@ -16,7 +16,7 @@
           "connector" : "kafka",
           "format" : "flexible-json",
           "properties.bootstrap.servers": "${PROPERTIES_BOOTSTRAP_SERVERS}",
-          "properties.group.id": "groupid",
+          "properties.group.id": "${PROPERTIES_GROUP_ID}",
           "scan.startup.mode" : "earliest-offset",
```

Comment:
this should be `group-offsets` (or omitted since that is the default). Otherwise, it will always start from the beginning.